schemas/littlehorse/service.proto

```diff
@@ -1423,8 +1423,13 @@ message GetLatestWfSpecRequest {
   optional int32 major_version = 2;
 }
 
-// The version of the LH Server according to Semantic Versioning
+// Get the current Server Version
 message ServerVersionResponse {
```

Comment:
Why is this change necessary? It's a breaking change and would break old `lhctl` versions who run `lhctl version` on a new server.